tests/EvmAcceptanceTests/test/Erc20.js

```diff
@@ -0,0 +1,93 @@
+const {expect} = require("chai");
+const {ethers} = require("hardhat");
+
+describe("ERC20 functionality", function () {
+  const TOTAL_SUPPLY = 1_000_000;
+  let contract;
+  before(async function () {
+    // The first three address needs to have funded with some zils
+    const [account1, account2, account3] = await ethers.getSigners();
+    const firstBalance = await ethers.provider.getBalance(account1.address);
+    const secondBalance = await ethers.provider.getBalance(account2.address);
+    const thirdBalance = await ethers.provider.getBalance(account3.address);
+    if (firstBalance == 0 || secondBalance == 0 || thirdBalance == 0) {
+      throw new Error("The first three accounts need to have some zils to run these tests.");
+    }
+
+    const Contract = await ethers.getContractFactory("ZEVMToken");
+    contract = await Contract.deploy("ZERC", "Zilliqa ERC Token", TOTAL_SUPPLY);
+  });
+
+  describe("General", function () {
+    it("Should return ZERC as the symbol of the token", async function () {
+      expect(await contract.symbol()).to.be.equal("ZERC");
+    });
+
+    it("Should return 1_000_000 as the total supply of the token", async function () {
+      expect(await contract.totalSupply()).to.be.equal(TOTAL_SUPPLY);
+    });
+
+    it("Should return 1_000_000 for the owner's balance at the beginning", async function () {
+      const [owner] = await ethers.getSigners();
+      expect(await contract.balanceOf(owner.address)).to.eq(TOTAL_SUPPLY);
+    });
+  });
+
+  describe("Transfer", function () {
+    it("Should be possible to transfer ZERC from the owner to another", async function () {
+      const [owner, receiver] = await ethers.getSigners();
+      const prevBalance = await contract.balanceOf(owner.address);
+
+      await contract.transfer(receiver.address, 1000);
+
+      expect(await contract.balanceOf(receiver.address)).to.eq(1000);
+      expect(await contract.balanceOf(owner.address)).to.eq(prevBalance - 1000);
+    });
+    // TODO: Add event related test(s), e.g. for Transfer event
+
+    // FIXME: In ZIL-4879
+    xit("Should not be possible to move more than available tokens to some address", async function () {
```

Comment:
xit? is that correct?

Reply:
@moboware yes. `xit` is for skipping tests. there are two failing tests which needs further debugging. I filed an issue for them. and added a FIXME comment with issue number for them.